README.md

```diff
@@ -1,2 +1,13 @@
 # Long read pipelines
-This repository contains long read processing pipelines (e.g. CCS error correction, alignment, de novo assembly, variant calling
+This repository contains pipelines for processing long read data from PacBio and/or Oxford Nanopore platforms.  Processing is designed to be consistent between technologies with platform-specific steps where required.
+
+The primary pipeline is LRWholeGenomeSingleSample, which performs the following actions:
```

Comment:
do you think this description should be local to the `LRWholeGenomeSingleSample.wdl`?
That way, when the WDL is changed, descriptions in the WDL should be updated accordingly as well, so that we don't need to update two files.

Reply:
Yeah - maybe we should just have a high-level README that describes what pipelines are present (and one line describing what each does), and then the *real* details at the beginning of the pipelines themselves?

Reply:
This is perfect.

Details are probably not necessary at the beginning, as the details are always changing fast, but I get what you mean.